lib/galaxy/datatypes/registry.py

```diff
@@ -821,6 +827,15 @@ def edam_formats( self ):
         return mapping
 
     @property
+    def edam_data( self ):
+        """
+        """
+        mapping = {}
+        for k, v in self.datatypes_by_extension.iteritems():
+            mapping[k] = v.edam_data
```

Comment:
Replace these 3 lines with:
```
mapping = dict((k, v.edam_data) for k, v in self.datatypes_by_extension.items())
```

(I'm suggesting `items()` instead of `iteritems()` here for future Python3 support)